egs/librispeech/ASR/prepare.sh

```diff
@@ -139,7 +139,7 @@ if [ $stage -le 3 ] && [ $stop_stage -ge 3 ]; then
   log "Stage 3: Compute fbank for librispeech"
   mkdir -p data/fbank
   if [ ! -e data/fbank/.librispeech.done ]; then
-    ./local/compute_fbank_librispeech.py
+    ./local/compute_fbank_librispeech.py --num-workers 4
```

Comment:
```suggestion
    ./local/compute_fbank_librispeech.py --num-workers $nj
```

we have `nj=15` at the beginning of this script.

Reply:
> we have `nj=15` at the beginning of this script.

Good catch. Updated to use the $nj variable instead of the hardcoded value 4 for better flexibility.